nucliadb_node/src/shards/shard_reader.rs

```diff
@@ -345,7 +347,21 @@ impl ShardReader {
                         shard_id: String::default(), // REVIEW: really?
                         prefix: Some(RelationPrefixSearchRequest {
                             prefix,
-                            ..Default::default()
+                            // Any type that is not Resource (avoid suggesting resource UUIDs)
+                            node_filters: vec![
+                                RelationNodeFilter {
+                                    node_type: NodeType::Entity.into(),
+                                    ..Default::default()
+                                },
+                                RelationNodeFilter {
+                                    node_type: NodeType::Label.into(),
+                                    ..Default::default()
+                                },
+                                RelationNodeFilter {
+                                    node_type: NodeType::User.into(),
+                                    ..Default::default()
+                                },
+                            ],
```

Comment:
I think that the expectation for this feature is that only entities are suggested.

Reply:
:thinking: That makes sense for me as well, should we confirm it with someone else? Or just change it to filter only entities?

Reply:
yup, good idea. let me confirm with @ebrehault

Reply:
As another data point: the current tests check that Users are also suggested, but I think it might be a mistake since the comments explicitly mention entities. It's probably just copypasting what the function returned as the expected value without double-checking.